modules/migration-rsync-root-non-root.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/installing-mtc.adoc
+// * migration_toolkit_for_containers/installing-mtc-restricted.adoc
+
+:_content-type: CONCEPT
+[id="migration-rsync-root-non-root]
+== Running Rsync as either root or non-root
+
+[IMPORTANT]
+====
+This section applies only when you are working with the OpenShift API, not the web console.
+====
+
+OpenShift environments have the `PodSecurityAdmission` controller enabled by default. This controller requires cluster administrators to enforce Pod Security Standards by means of namespace labels. All workloads in the cluster are expected to run one of the following Pod Security Standard levels: `Privileged`, `Baseline` or `Restricted`. Every cluster has its own default policy set.
+
+To guarantee successful data transfer in all environments, {mtc-full} ({mtc-short}) 1.7.5 introduced changes in Rsync pods, including running Rsync pods as non-root user by default. This ensures that data transfer is possible even for workloads that do not necessarily require higher privileges. This change was made because it is best to run workloads with lowest level of privileges possible.
+
+== Manually overriding default non-root operation for data transfer
```

Comment:
Is this content copy/pasted verbatim from modules/migration-rsync-manually-overriding.adoc? Consider applying mod docs principles to not duplicate the content.